ilastik/plugins.py

```diff
@@ -161,17 +161,19 @@ def checkFilesExist(self, filename):
         ''' Check whether the files we want to export (when appending the base filename) are already present '''
         return False
 
-    def export(self, filename, hypothesesGraph, *, objectFeaturesSlot, labelImageSlot, rawImageSlot, bdvFilepathSlot):
+    def export(self, filename, hypothesesGraph, pluginExportContext):
         """Export the tracking solution stored in the hypotheses graph's "value" and "divisionValue"
         attributes (or the "lineageId" and "trackId" attribs). See https://github.com/chaubold/hytra for more details.
 
         :param filename: string of the file where to save the result (or folder where to put the export files)
         :param hypothesesGraph: hytra.core.hypothesesgraph.HypothesesGraph filled with a solution
-        :param objectFeaturesSlot (lazyflow.graph.InputSlot): connected to the RegionFeaturesAll
+        :param pluginExportContext (ilastik.plugins.PluginExportContext): instance of PluginExportContext containing data necessary for the export such as:
+            - objectFeaturesSlot (lazyflow.graph.InputSlot): connected to the RegionFeaturesAll
             output of ilastik.applets.trackingFeatureExtraction.opTrackingFeatureExtraction.OpTrackingFeatureExtraction
-        :param labelImageSlot (lazyflow.graph.InputSlot): labeled image slot
-        :param rawImageSlot (lazyflow.graph.InputSlot): raw image slot
-        :param bdvFilepathSlot (lazyflow.graph.InputSlot): BigDataViewer file path slot
+            - labelImageSlot (lazyflow.graph.InputSlot): labeled image slot
+            - rawImageSlot (lazyflow.graph.InputSlot): raw image slot
+            - additionalPluginArgumentsSlot (lazyflow.graph.InputSlot): slot containing a dictionary
+            with plugin specific arguments
```

Comment:
we should check if that renders properly, or if that has to be indented

Reply:
it doesn't render correctly. Nice catch!

Reply:
:white_check_mark: